contrib/utilities/indent

```diff
@@ -44,3 +44,22 @@ fi
 # with up to 10 in parallel
 find tests include source examples \( -name '*.cc' -o -name '*.h' \) -print | xargs -n 50 -P 10 astyle --options=contrib/utilities/astyle.rc
 
+# format .inst.in files. We need to replace \{ and \} because it confuses
+# astyle.
+format_inst() {
+f=$1;
+cp $f $f.tmp
+sed -i 's#\\{#{ //#g' $f.tmp
+sed -i 's#\\}#} //#g' $f.tmp
+astyle --quiet $f.tmp
+sed -i 's#{ //#\\{#g' $f.tmp
+sed -i 's#} //#\\}#g' $f.tmp
+if ! diff -q $f $f.tmp >/dev/null
+then
+  cp $f.tmp $f
+fi
+rm $f.tmp
```

Comment:
The ```.tmp``` file is needed to not unnecessarily update mtime on all .inst files I guess.